src/index.html

```diff
@@ -22,6 +22,348 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div>
+      <header class="header">
+        <a href="#">
+          <img
+            class="img"
+            src="/src/images/logo.png"
+            alt="MOYO logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                class="nav__link is-active"
+                href="#"
```

Comment:
The `data-qa` attribute should be added to the 4th nav link (`Laptops & Computers`) as `data-qa="nav-hover"` for testing purposes, as per the task requirements.